docs/dev.md

```diff
@@ -180,3 +180,4 @@ These parameters are available in addition to the ones in the `dev` section abov
 | dockerfile | Location of a Dockerfile to be used by dev mode to build the Docker image for the container that will run your Liberty server.  The directory containing the Dockerfile will also be the context for the `docker build`. The default value is `Dockerfile`. | No |
 | dockerBuildTimeout | Maximum time to wait (in seconds) for the completion of the Docker operation to build the image. The value must be an integer greater than 0. The default value is `60` seconds. | No |
 | skipDefaultPorts | If set to `true`, dev mode will not publish the default Docker port mappings of `9080:9080` (HTTP) and `9443:9443` (HTTPS). Use this option if you would like to specify alternative local ports to map to the exposed container ports for HTTP and HTTPS using the `dockerRunOpts` parameter. | No |
+| keepTempDockerfile | If set to `true`, dev mode will not delete the temporary file which is the Dockerfile used to build the container. This file is handy in case you need to debug the process of building the container. The name of the temporary file can be seen when dev mode displays the `docker build` command. | No |
```

Comment:
Also mention that the default value is `false`

Reply:
ok